app/controllers/comment_controller.php

```diff
@@ -0,0 +1,55 @@
+<?php
+class CommentController extends AppController
+{
+	CONST MAX_ITEM_PER_PAGE = 10;
+
+	public function view()
+	{
+		session_start();
```

Comment:
also, shouldn't you be checking for sessions if the user is currently logged in or not? currently, the user can still access this page even if he/she is not logged in.